docs/api/cases/cases-api-get-comments.asciidoc

```diff
@@ -61,20 +61,20 @@ The API returns the requested comment JSON object. For example:
 [source,json]
 --------------------------------------------------
 {
-  "id":"8acb3a80-ab0a-11ec-985f-97e55adae8b9",
-  "version":"Wzc5NzYsM10=",
-  "comment":"Start operation bubblegum immediately! And chew fast!",
+  "id":"8048b460-fe2b-11ec-b15d-779a7c8bbcc3",
+  "version":"WzIzLDFd",
   "type":"user",
   "owner":"cases",
-  "created_at":"2022-03-24T00:37:10.832Z",
-  "created_by": {
-    "email": "classified@hms.oo.gov.uk",
-    "full_name": "Classified",
-    "username": "M"
-    },
-  "pushed_at": null,
-  "pushed_by": null,
-  "updated_at": null,
-  "updated_by": null
+  "comment":"A new comment",
+  "created_at":"2022-07-07T19:32:13.104Z",
+  "created_by":{
```

Comment:
It would be nice to use a more interesting example here, such as by setting a full name and email for the user, and possibly having updated the comment or pushed to an external connector.

Reply:
Yes, though I'm trying to balance that with the ease of making these work in our new automated example tester. So I'm inclined to start simple until we get adept with it, if that's okay.

Reply:
OK! Starting simple sounds good to me.